site/src/pages/components/popover-hint.research.explainer.mdx

```diff
@@ -0,0 +1,169 @@
+---
+menu: Proposals
+name: Popover=hint and Hover Triggering (Explainer)
+path: /components/popover-hint.research.explainer
+pathToResearch: /components/popup.research
+layout: ../../layouts/ComponentLayout.astro
+---
+
+# Popover Hover Triggering and `popover=hint`
+
+- [@mfreed7](https://github.com/mfreed7), [@scottaohara](https://github.com/scottaohara), [@aleventhal](https://github.com/aleventhal)
+- June 8, 2023
+
+{/* <!-- START doctoc generated TOC please keep comment here to allow auto update --> */}
+{/* <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE --> */}
+## Table of Contents
+
+- [Background](#background)
+- ["Hover" triggering](#hover-triggering)
+  - [Add the `hover` value](#add-the-hover-value)
+  - [CSS values to control delay](#css-values-to-control-delay)
+  - [Popover types](#popover-types)
+  - [Example](#example)
+- [`popover=hint` Behavior](#popoverhint-behavior)
+- [Accessibility](#accessibility)
+  - [Why differentiate between rich and plain hints?](#why-differentiate-between-rich-and-plain-hints)
+  - [A11y API support](#a11y-api-support)
+- [`::tooltip` pseudo element](#tooltip-pseudo-element)
+- [Monkeypatch spec](#monkeypatch-spec)
+- [Articles and References](#articles-and-references)
+
+{/* <!-- END doctoc generated TOC please keep comment here to allow auto update --> */}
+
+
+# Background
+
+The [Popover API](https://html.spec.whatwg.org/multipage/C#the-popover-attribute) ([MDN](https://developer.mozilla.org/en-US/docs/Web/API/Popover_API), [OpenUI Explainer](https://open-ui.org/components/popover.research.explainer/)) is now part of the HTML standard. It allows developers to create popovers of varying types and also easily trigger them from buttons, without using Javascript.
+
+This explainer discusses two important extensions of the Popover API:
+
+1. The ability to trigger popovers via a "hover" type interaction.
+2. The ability to show a new type of popover for "hints".
+
+These two capabilities are orthogonal but closely related, and this explainer therefore discusses them both.
+
+The motivating use case, but certainly not the only use case, for these two capabilities is in the construction of "tooltips". Typically, tooltips are transient bits of additional information related to a bit of content. This additional information is *not required* and is supplementary in nature. Usually the user accesses this tooltip by hovering, keyboard-focusing, or long-pressing a control.
+
+# "Hover" triggering
```

Comment:
I agree - `hover` is misleading in that it applies only pointer compatibility. Another alternative might be `active`?

Reply:
I agree with @keithamus and @iansan5653 for keyboard users the popup should still appear upon traversal to the element that has the associated popup. That said, I'm curious of @scottaohara thoughts as that may produce numerous popups while tabbing through a site.

Reply:
it is still a sort of open question about this that I think may need to be brain stormed a bit.

For instance, in the scope of tooltips it may be better to think about introducing a user setting or alternate key to show/hide a tooltip at will.  Esc is often used to dismiss, but the only way to get a tooltip back is to navigate away and then back to the element.  That's silly and we could do better with this.  

But there are other types of UI - e.g., text/search input fields with autocomplete popups that one might want to show up immediately on focus, and it'd be good if we could support both.

Reply:
Let's move at least the bikeshedding part of this discussion (which is great, thanks all!) over to this issue: https://github.com/openui/open-ui/issues/767

Scott, we can open another issue to discuss your comments about a user setting or alternate key - does that make sense?